src/tasks/LibraryBuilder/LibraryBuilder.cs

```diff
@@ -66,6 +66,26 @@ public bool IsSharedLibrary
         }
     }
 
+    /// <summary>
+    /// Determines whether or not the mono runtime auto initialization
+    /// tremplate, autoinit.c, is used.
+    /// </summary>
+    public bool UsesCustomRuntimeInitCallback { get; set; }
+
+    public string? AssembliesLocation { get; set; }
+
+    /// <summary>
+    /// </summary>
+    public ITaskItem[] AppContextKeys { get; set; } = Array.Empty<ITaskItem>();
+
+    /// <summary>
+    /// </summary>
+    public ITaskItem[] AppContextValues { get; set; } = Array.Empty<ITaskItem>();
+
+    /// <summary>
+    /// </summary>
+    public string? RuntimeConfigBinFile { get; set; }
```

Comment:
I don't think this is needed any longer.

Reply:
Thanks for the catch!